File: narrowartgallery/template/narrowartgallery.cpp
#include <iostream>
#include <string>
#include <vector>
using namespace std;

int solve(vector<vector<int>> &rooms, int num_to_close_off)
{
	//YOUR CODE HERE
	return -1;
}

int main(){
	int N, k;
	cin >> N >> k;
	vector<vector<int>> rooms;
	int l, r;
	for (int i=0; i<N; i++)
       	{
		cin >> l >> r;
		vector<int> row;
		row.push_back(l);
		row.push_back(r);
		rooms.push_back( row );
	}

	cout << solve(rooms, k) << endl;
	return 0;
}

File: narrowartgallery/template/narrowartgallery.py
def solve(rooms:list, num_to_close_off: int) -> int:

    # YOUR CODE HERE

    return 0

if __name__ == '__main__':

    N, k = map(int,input().split(' '))
    rooms = []
    for _ in range(N):
        rooms.append( list(map(int,input().split(' '))) )
    print( solve(rooms, k) )
